Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -678,6 +679,7 @@ BrowserWindow::BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::Cook
     m_new_tab_button_toolbar->setMovable(false);
     m_new_tab_button_toolbar->setStyleSheet("QToolBar { background: transparent; }");
     m_new_tab_button_toolbar->setIconSize(QSize(16, 16));
+    m_tabs_container->setCornerWidget(m_new_tab_button_toolbar, Qt::TopRightCorner);
```

Comment:
If approaching it like this, I think you should be able to remove the width reduction. (See https://github.com/LadybirdBrowser/ladybird/issues/768)

Reply:
Yeah I assumed it would, but it's still leaving a gap. I'm right now trying a way of manually drawing the tab-bar background in the gap.